### DISCUSSION TITLE
Using fzf-lua to examine built-in options

### DISCUSSION THREAD

**Discussion by danielfleischer** [LazyVim/LazyVim#5072] [Q&A]
Is there a way/recipe to use fzf-lua to examine nvim options? (telescope had a way of doing that.)

**Comment by windowsrefund** [comment]
I don't think so but you can run `:let` to see everything. The results won't be immediately "usable" since they'll be displayed in a notice. However, you can then use `<leader>n` to look at your Notification History. Not the best answer but it's the closest thing I've come up with in order to (almost) accomplish what the old `:Telescope vim_options` used to provide.

**Reply by danielfleischer** [reply]
Yes, no fuzzy though.
